### PR TITLE
(AP-7) [codegen.yml, package.json, webpack.config, asyncActions(user), .gitignore, tests, types]

### PR DESCRIPTION
**Pull request by DenisJankelaic**
- Added magento backend url variable from .env to codegen config file.
- Added removeCart action on sign out.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -33,31 +36,23 @@ module.exports = async env => {
             'informed',
             'react',
             'react-dom',
-            'react-feather',
```
Comment (AndriusUkelis) [inline webpack.config.js:36]: Nereikia išimt feather. Palik

File: webpack.config.js [hunk 2]
```diff
@@ -33,31 +36,23 @@ module.exports = async env => {
...
             'react-redux',
             'react-router-dom',
             'redux',
             'redux-actions',
             'redux-thunk',
             'react-bootstrap'
         ],
-        special: {
-            'react-feather': {
```
Comment (AndriusUkelis) [inline webpack.config.js:45]: Same

**Review by AndriusUkelis — CHANGES_REQUESTED** [review]

File: src/lib/types/graphql-types.d.ts [hunk 3]
```diff
@@ -1,12 +1,8740 @@
 /* eslint-disable */
```
Comment (AndriusUkelis) [inline src/lib/types/graphql-types.d.ts:1]: Jei šitas failas yra autogenerated, tai kam jį commitint?

Comment (DenisJankelaic) [reply src/lib/types/graphql-types.d.ts:1]: Neturėjo susicommitint. Tuoj patvarkysiu.

**Review by AndriusUkelis — APPROVED** [review]

**Review by mraivaras — APPROVED** [review]

**Pull request merged**